server/export.go

```diff
@@ -76,6 +76,7 @@ func ExportCmd(appExporter types.AppExporter, defaultNodeHome string) *cobra.Com
 
 			doc.AppState = appState
 			doc.Validators = validators
+			doc.InitialHeight = appHeight + 1
```

Comment:
Not saying this is necessarily incorrect, but why are we returning `+1` here? AFAIU, `InitialHeight` (when non-zero) will be used as the height to load during `InitChain`. If it's `+1`, we won't have that height to load, correct?

Reply:
Ah, I saw https://github.com/cosmos/iavl/pull/299 linked in the issue, my understanding was that before InitChain, we would need to dump the exported genesis into a new store at version `N+1`, and then start the chain from there.

Reply:
Maybe I'm missing something, but I assumed during `InitChain`, we load `InititalHeight` from the commit multi-store. In any case, have you tested this out?

Reply:
yeah, that would also make sense. I'm working rn on InitChain, will try out both cases.